### PR TITLE
NanoVG example updates, better SVG support.

### PR DESCRIPTION
**Pull request by SmilyOrg**
This also changes the API somewhat so that `svg` is now the first parameter for `drawSVG` and not the last. This is easily fixable in client code so I think it should be acceptable to avoid future trouble when adding more parameters.

Updated NanoVG example to work with the changes and added the swirly spiral.
Added the ability to get the width and height of the SVG.
Added line width support to SVG and a lineThickness multiplier on draw.

### CODE REVIEW

File: sdk/src/loom2d/Display/Graphics.ls [hunk 1]
```diff
@@ -104,6 +104,8 @@ package loom2d.display
      */
     public native class SVG
     {
+        public native var width:Number;
```
Comment (bengarney) [inline sdk/src/loom2d/Display/Graphics.ls]: Document members?

File: sdk/src/loom2d/Display/Graphics.ls [hunk 2]
```diff
@@ -224,9 +226,9 @@ package loom2d.display
         public native function drawTextBox(x:Number, y:Number, width:Number, text:String):void;
         
         /**
-         * Draw an SVG at the given center point.
+         * Draw an SVG at the given point.
          */
-        public native function drawSVG(x:Number, y:Number, scale:Number, svg:SVG):void;
+        public native function drawSVG(svg:SVG, x:Number, y:Number, scale:Number = 1, lineThickness:Number = 1):void;
```
Comment (bengarney) [inline sdk/src/loom2d/Display/Graphics.ls]: Can you document the params more clearly with @param markups?

File: docs/examples/NanoVG/src/NanoVG.ls [hunk 3]
```diff
@@ -444,16 +468,43 @@ package
...
-                g.lineStyle(3 + 1*Math.sin(1000*i/(n-1)*Math.TWOPI + t*50), hsvToRgb((i/n*360*3 + t*200)%360, 1, 1), 1);
+                //var ang = (i/(n-1)*spiral+t*0.5+rot) * Math.TWOPI;
+                var ang = spiral*i+rot*i/n;
+                //var ang = 1/sb*Math.log(r/sa)+rot;
+                g.lineStyle(3 + 1*Math.sin(1000*i/(n-1)*Math.TWOPI + t*50), hsvToRgb((i/n*360*3 + t*200)%360, 0.8, 1), 1);
                 g.lineTo(cx+Math.cos(ang)*r, cy+Math.sin(ang)*r);
             }
             //*/
+            
+            /*
```
Comment (bengarney) [inline docs/examples/NanoVG/src/NanoVG.ls]: What does this test?

**Comment by bengarney** [comment]
Nitted. Looks good otherwise.

**Comment by SmilyOrg** [comment]
Your nits are now outdated!

**Pull request merged**